wallet/Cargo.toml

```diff
@@ -14,6 +14,7 @@ bip39 = { path = "../bip39" }
 chain-path-derivation = { path = "../chain-path-derivation" }
 hdkeygen = { path = "../hdkeygen" }
 hex = "0.4.2"
+im-rc = "15.0.0"
```

Comment:
I think we can remove this deps. 
```suggestion
```

Reply:
why though? I mean, we are still using it in the utxo store

Reply:
ha yes, my bad, I thought we had moved on to something else.